.teamcity/components/generated/services.kt

```diff
@@ -97,5 +97,6 @@ var services = mapOf(
         "trafficmanager" to "Traffic Manager",
         "vmware" to "VMware",
         "videoanalyzer" to "Video Analyzer",
-        "web" to "Web"
+        "web" to "Web",
+        "webpubsub" to "Webpubsub"
```

Comment:
could we use a property description here?
```suggestion
        "webpubsub" to "Web Publishing Subscription"
```